biz.aQute.launcher/src/aQute/launcher/Launcher.java

```diff
@@ -471,15 +480,39 @@ private int activate() throws Exception {
 		return result;
 	}
 
-	private boolean isImmediate(BundleActivator activator) {
+	private boolean isAfterFrameworkStart(BundleActivator activator) {
```

Comment:
This is still a confusing name. The FrameworkEvent.STARTED event is defined to mean:

```
The Framework has started.

This event is fired when the Framework has started after all installed bundles that are marked to be started have been started and the Framework has reached the initial start level. 
```

but you really mean `AFTER_FRAMEWORK_INIT`. That is, after the framework is initialized by Framework.init and before any bundles are installed and started.

Reply:
That is true. I had this thought at some point as well. I will change the value and method name to correspond to  ```AFTER_FRAMEWORK_INIT```